libs/ui-watt/src/lib/components/drawer/watt-drawer.component.ts

```diff
@@ -38,33 +38,27 @@ export class WattDrawerComponent {
   @Output()
   closed = new EventEmitter<void>();
 
+  /** @ignore */
+  @ContentChild(WattDrawerTopBarDirective)
+  topBar?: WattDrawerTopBarDirective;
+
   /** @ignore */
   @ContentChild(WattDrawerContentDirective)
   content?: WattDrawerContentDirective;
 
+  /** @ignore */
+  @ViewChild('topBarVcr', { read: ViewContainerRef, static: false })
+  private topBarVcr?: ViewContainerRef;
+
   /** @ignore */
   @ViewChild('contentVcr', { read: ViewContainerRef, static: false })
   private contentVcr?: ViewContainerRef;
 
   /** @ignore */
-  private _isOpened = false;
+  private areViewsCreated = false;
```

Comment:
Question: why do we need this, when we have the `opened` property?

Reply:
Good question. We can only use `opened` to prevent the drawer from opening again. Will make the changes.